### PR TITLE
Add documentation on configuring Kafka

### PR DESCRIPTION
**Pull request by agrare**

### CODE REVIEW

File: _includes/configuration.md [hunk 1]
```diff
@@ -96,6 +96,33 @@ necessary to alter the `max_connections` setting.
...
+1.  Start the appliance and open a terminal console.
+
+2.  Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
+    summary screen displays.
+
+3.  Press **Enter** to manually configure settings.
+
+4.  Select **Configure Application** from the menu.
+
+5. Select **Make No Database Changes** from the menu.
```
Comment (Fryguy) [inline _includes/configuration.md:112]:
```suggestion
Super minor but the alignment is weird

1. Start the appliance and open a terminal console.

2. Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
    summary screen displays.

3. Press **Enter** to manually configure settings.

4. Select **Configure Application** from the menu.

5. Select **Make No Database Changes** from the menu.
```

File: _includes/configuration.md [hunk 2]
```diff
@@ -96,6 +96,33 @@ necessary to alter the `max_connections` setting.
...
+6. You can either configure the current appliance as a kafka broker, or point the
+   appliance to an existing external kafka broker.
+
+   Select the appropriate option either
+   **Configure this appliance as a messaging server** or
+   **Connect to an external messaging system**
+   to connect to an external kafka broker.  You will be asked to fill in the
+   required **Message Client Parameters** like IP address and username/password.
+
+7. Select Proceed and appliance_console will apply the configuration that you have
```
Comment (Fryguy) [inline _includes/configuration.md:123]:
```suggestion
7. Select **Proceed** and appliance_console will apply the configuration that you have
```

**Review by Fryguy — APPROVED** [review]

File: _includes/configuration.md [hunk 3]
```diff
@@ -96,6 +96,33 @@ necessary to alter the `max_connections` setting.
...
 {{ site.data.product.title_short }} will then configure the external database.
 
+#### Configuring a Kafka broker
+
+Some features of ManageIQ can be enabled by enabling a message server.
+
+1. Start the appliance and open a terminal console.
+
+2. Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
+    summary screen displays.
```
Comment (bdunne) [inline _includes/configuration.md:106]: What do you think of this:
```suggestion
2. Run the `appliance_console` command and the summary screen will be displayed.
```
I don't feel like the substitution adds a lot of value.

Comment (agrare) [reply _includes/configuration.md:106]: :+1:

**Pull request merged**